src/displayapp/screens/PineTimeStyle.cpp

```diff
@@ -0,0 +1,294 @@
+#include "PineTimeStyle.h"
+
+#include <date/date.h>
+#include <lvgl/lvgl.h>
+#include <cstdio>
+#include "BatteryIcon.h"
+#include "BleIcon.h"
+#include "NotificationIcon.h"
+#include "Symbols.h"
+#include "components/battery/BatteryController.h"
+#include "components/ble/BleController.h"
+#include "components/ble/NotificationManager.h"
+#include "components/heartrate/HeartRateController.h"
+#include "components/settings/Settings.h"
+#include "../DisplayApp.h"
+
+/*
```

Comment:
You might just want to use GPLv3 license header and add this comment to its comment section. Would look a bit nice.

Reply:
Usually placed before any includes